### PR TITLE
Add spurt (Addon)

### PR DESCRIPTION
**Pull request by Ultradragon005**
From: https://discord.com/channels/1094344790910455908/1262819630452314252
--
The addon works but gonna add some feature before complete, and has good default settings after some testing.

### CODE REVIEW

**Comment by HyperAtill** [comment]
Better name: "Dynamo But Reversed"

**Comment by Ultradragon005** [comment]
basically yeah 😂

**Comment by HyperAtill** [comment]
Drakos is the new EHR developer....

**Comment by Ultradragon005** [comment]
It's finished

File: Resources/Lang/en_US.json [hunk 1]
```diff
@@ -3978,6 +3978,15 @@
   "AADelay": "Time to Leave Device",
   "AAWarning": "<#ffff00>You must move away from this device or you die!</color>",
   "AADone": "The process has finished",
+  "Spurt": "Spurt",
+  "Prefix.Spurt": "Spurt",
+  "SpurtInfo": "Boosts Movement",
+  "SpurtInfoLong": "\\nWhen starting to walk you gain an enormous speed boost, which swiftly deteriorates, untill you have to rest still for a while to rejuvenate your speed.",
```
Comment (Gurge44) [inline Resources/Lang/en_US.json:3984]: 💀

Comment (Ultradragon005) [reply Resources/Lang/en_US.json:3984]: what's wrong 😼

Comment (Gurge44) [reply Resources/Lang/en_US.json:3984]: "untill"

Comment (Gurge44) [reply Resources/Lang/en_US.json:3984]: Whatever I'll fix it

Comment (Ultradragon005) [reply Resources/Lang/en_US.json:3984]: oh

**Pull request merged**